src/map/ai/states/mobskill_state.cpp

```diff
@@ -131,3 +144,9 @@ void CMobSkillState::Cleanup(time_point tick)
         m_PEntity->loc.zone->PushPacket(m_PEntity, CHAR_INRANGE, new CActionPacket(action));
     }
 }
+
+// TODO move this to lua/sql
```

Comment:
Yeah, for sure we can add to/use/reuse `mob_skill_flag` for this in mob_skills.sql

Reply:
Just add yourself one with the value of `0x40` for now